.devcontainer/devcontainer.json

```
@@ -0,0 +1,20 @@
+
+{
+  "image": "rapidsai/devcontainers:23.06-cpp-gcc12-cuda12.1-ubuntu22.04",
+
+  "hostRequirements": { "gpu": true },
+
+  "initializeCommand": ["/bin/bash", "-c", "mkdir -p .cache/.{aws,config}"],
```

We should be consistent

```suggestion
  "initializeCommand": [
    "/bin/bash",
    "-c",
    "mkdir -p .cache/.{aws,config}"
  ],
```